metadata-app/src/api/index.test.js

```diff
@@ -2,10 +2,7 @@ import moxios from 'moxios';
 import Api from '.';
```

Comment:
It's a bit hard to understand the changes in this file, but tried to review, would be nice if we could have a separate PR